source/Sale.h

```diff
@@ -18,6 +18,46 @@ PARTICULAR PURPOSE.  See the GNU General Public License for more details.
 
 #include <set>
 #include <string>
+#include <algorithm>
+
+
+template <class Item>
+class OutfitSale;
+
+
+
+// Container used for the price and selling type of different items (outfits in this case).
+class Sold {
+public:
+	enum ShowSold {
+		NONE = 0,
+		DEFAULT = (1 << 0),
+		IMPORT = (1 << 1),
+		HIDDEN = (1 << 2)
+	};
+	
+	
+public:
+	const double GetCost() const;
```

Comment:
for std::string getshown it is needed because that string is a const but for the rest you're right I guess, not same conventions they bashed my head with in my university ^^ will remove

Reply:
In getshow it's a reference so it's ok :) Just for when you're not returning a reference.